docs/en/ingest-management/elastic-agent/install-fleet-managed-agent.asciidoc

```diff
@@ -0,0 +1,29 @@
+["appendix",role="exclude",id="install-fleet-managed-agent"]
+= Install a {fleet} managed {agent}
+
+This guide describes how to:
+
+* Install an {agent} that will be managed with {fleet}
+* Enroll the {agent} with {fleet}
+
+If you have come here from the {agent} download page, then you are in the right place.
+
+[discrete]
+[[minimal-add-agent-to-fleet]]
+== Step 1: Add an {agent} to {fleet}
+
+{agent} is a single, unified agent that you can deploy to hosts or containers to
```

Comment:
This is OK for now, but wanted to let you know that we are going to update this description to encompass data protection.

The description in the overview is a starting point, but I think it needs to be more concise: https://www.elastic.co/guide/en/fleet/current/fleet-overview.html#elastic-agent

Just bringing this up in case you want to tackle it.... otherwise I can fix this when I make the change in other places.

Reply:
Agreed, I looked at the product page for Agent to see what to say about protecting endpoints and did not find it.  As I build the endpoint security getting started guide I may work out wording.  I will also check the video from training.